src/server/protocol.ts

```diff
@@ -1947,6 +1948,17 @@ namespace ts.server.protocol {
         body?: CompletionEntry[];
     }
 
+    export interface CompletionInfoResponse extends Response {
+        body?: CompletionInfo;
+    }
+
+    export interface CompletionInfo {

```

Comment:
i would add a comment on this and on Completions to state the difference and ask API users to use the new command moving forward.